developer-experience/readme.md

```diff
@@ -0,0 +1,203 @@
+# Developer Experience
+
+Developer experience refers to the how easy or difficult it is for a developer to perform essential tasks needed to implement a change. A positive developer experience would mean these tasks are relatively easy for the team (see measures below).
+
+The essential tasks are identified below.

```

Comment:
Not sure if this is in scope here or not, but do you see things like "finding the backlog" or "understanding the board" as falling into this proposed fundamental?

Reply:
There is a section in the playbook for backlog management that i believe would be responsible for what you are calling out here. Agree though it definitely can create churn if work is hard to find or ambiguous.